exercises/practice/variable-length-quantity/.meta/test_template.tera

```diff
@@ -19,7 +19,7 @@ fn {{ test.description | slugify | replace(from="-", to="_") }}() {
             0x{{ byte | to_hex }},
         {%- endfor -%}
     ];
-    let output = vlq::{{ fn_names[1] }}(input);
+    let output = vlq::from_bytes(input);
```

Comment:
`fn_names[1]` was failing for some reason, the explicit function name is more readable anyway.